src/pages/Download.js

```diff
@@ -1,9 +1,238 @@
-import { SectionTitle } from '../common/Section'
+import {
+  GridSection,
+  GridSectionTitle,
+  Section,
+  SectionTitle,
+  WideHSection,
+} from '../common/Section'
```

Comment:
Could you remove those unused components from import?

![image](https://user-images.githubusercontent.com/2367032/140625585-44cb68b4-4a7f-41a2-a919-908a04aaac79.png)

Reply:
Done